kommunicate/src/main/java/io/kommunicate/zendesk/KmZendeskClient.java

```diff
@@ -305,7 +309,11 @@ public void run() {
                     listOfMessage = Arrays.asList(kmConversationResponse.getMessage());
                     Collections.reverse(listOfMessage);
                     for(Message message : listOfMessage) {
-                        String username;
+                        if (Message.GroupMessageMetaData.FALSE.getValue().equals(message.getMetaDataValueForKey(Message.GroupMessageMetaData.KEY.getValue()))) {
+                            continue;
```

Comment:
@amntoppo what is this check?

Reply:
```
        KEY("show"),
        HIDE_KEY("hide"),
        FALSE("false"),
        TRUE("true");
```
The first message contains `"show": "false"`

We are checking if the message contains show: false